src/redux/books/books.js

```diff
@@ -0,0 +1,28 @@
+const ADD = 'bookstore/books/addBook';
+const REMOVE = 'bookstore/books/removeBook';
+
+const initialState = {
+  books: [],
+};
+
+export default function booksReducer(state = initialState, action) {
+  switch (action.type) {
+    case ADD:
+      return { ...state };
```

Comment:
- [ Required ] I think that this statement would not ensure that a new book is added, remember that when `ADD` book action is dispatched a new book should be added to the redux state. To ensure that a new book is added, please consider refactoring this code as shown below
```JavaScript
  return [ ...state, action.payload ]
```